### ISSUE TITLE
Basic Authentication Example?

### ISSUE THREAD

**Issue by pcj** [andygrunwald/go-jira#120]
Reading the docs I tried the following:

```go

import (
	"strings"

	"github.com/urfave/cli"
	jira "gopkg.in/andygrunwald/go-jira.v1"
)

func NewJiraClientFromCliContext(c *cli.Context) (*jira.Client, error) {
	tp := jira.BasicAuthTransport{
		Username: c.GlobalString("jira_user"),
		Password: c.GlobalString("jira_password"),
	}

	client, err := jira.NewClient(tp.Client(), strings.TrimSpace(c.GlobalString("jira_url")))
	if err != nil {
		return nil, err
	}
	return client, nil
}
```

But this does not compile.  it seems the `jira.BasicAuthTransport` does not exist.  However, I see this test:

```go

func TestClient_NewRequest_BasicAuth(t *testing.T) {
	c, err := NewClient(nil, testJIRAInstanceURL)
	if err != nil {
		t.Errorf("An error occurred. Expected nil. Got %+v.", err)
	}

	c.Authentication.SetBasicAuth("test-user", "test-password")

	inURL := "rest/api/2/issue/"
	inBody := &Issue{Key: "MESOS"}
	req, err := c.NewRequest("GET", inURL, inBody)

	if err != nil {
		t.Errorf("An error occurred. Expected nil. Got %+v.", err)
	}

	username, password, ok := req.BasicAuth()
	if !ok || username != "test-user" || password != "test-password" {
		t.Errorf("An error occurred. Expected basic auth username %s and password %s. Got username %s and password %s.", "test-user", "test-password", username, password)
	}
}
```

Which one is correct?  Are docs out of date?  Am I missing something here?

**Comment by rbriski** [comment]
Don't use `SetBasicAuth`, it's deprecated.  The `BasicAuthTransport` struct is here:
https://github.com/andygrunwald/go-jira/blob/master/jira.go#L289

Are you sure you're on the latest version?

Can you try going to `examples/basicauth` and doing `go run main.go`?

You can look at the example code here:
https://github.com/andygrunwald/go-jira/blob/master/examples/basicauth/main.go

**Comment by rbriski** [comment]
Oh, I see the issue.  You're using `gopkg.in` and I don't think I've versioned the code to correctly keep up with it.  Let me do that now.

**Comment by rbriski** [comment]
OK, I've released `v1.1.0` which should update the `gopkg.in` tag.  Can you `go get` again?

**Comment by pcj** [comment]
Party on Garth, it works!

Thanks for quick response.
